When regenerating the bindings, bool.cpp is not generated anymore
--- a/wscript	2023-06-07 03:31:16.000000000 +0200
+++ b/wscript	2024-09-02 16:31:16.176552571 +0200
@@ -570,7 +570,6 @@ def build(bld):
         features = 'c cxx cshlib cxxshlib pyext',
         target   = makeTargetName(bld, 'siplib'),
         source   = ['sip/siplib/apiversions.c',
-                    'sip/siplib/bool.cpp',
                     'sip/siplib/descriptors.c',
                     'sip/siplib/int_convertors.c',
                     'sip/siplib/objmap.c',
